lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -145,59 +145,7 @@ export class AuthorizationCodeClient extends BaseClient {
         const requestBody = await this.createTokenRequestBody(request);
         const headers: Record<string, string> = this.createDefaultTokenRequestHeaders();
 
-        return this.executePostToTokenEndpoint(authority.tokenEndpoint, requestBody, headers, thumbprint);
-    }
-
-    /**
-     * Generates a map for all the params to be sent to the service
-     * @param request
-     */
-    protected async createTokenRequestBody(request: AuthorizationCodeRequest): Promise<string> {
-        const parameterBuilder = new RequestParameterBuilder();
-
-        parameterBuilder.addClientId(this.config.authOptions.clientId);
-
-        // validate the redirectUri (to be a non null value)
-        parameterBuilder.addRedirectUri(request.redirectUri);
-
-        // Add scope array, parameter builder will add default scopes and dedupe
-        parameterBuilder.addScopes(request.scopes);
-
-        // add code: user set, not validated
-        parameterBuilder.addAuthorizationCode(request.code);
-
-        // add code_verifier if passed
-        if (request.codeVerifier) {
-            parameterBuilder.addCodeVerifier(request.codeVerifier);
-        }
-
-        if (this.config.clientCredentials.clientSecret) {
-            parameterBuilder.addClientSecret(this.config.clientCredentials.clientSecret);
-        }
-
-        if (this.config.clientCredentials.clientAssertion) {
-            const clientAssertion = this.config.clientCredentials.clientAssertion;
-            parameterBuilder.addClientAssertion(clientAssertion.assertion);
-            parameterBuilder.addClientAssertionType(clientAssertion.assertionType);
-        }
-
-        parameterBuilder.addGrantType(GrantType.AUTHORIZATION_CODE_GRANT);
-        parameterBuilder.addClientInfo();
-
-        if (request.authenticationScheme === AuthenticationScheme.POP && !!request.resourceRequestMethod && !!request.resourceRequestUri) {
-            const popTokenGenerator = new PopTokenGenerator(this.cryptoUtils);
-            const cnfString = await popTokenGenerator.generateCnf(request.resourceRequestMethod, request.resourceRequestUri);
-            parameterBuilder.addPopToken(cnfString);
-        }
-
-        const correlationId = request.correlationId || this.config.cryptoInterface.createNewGuid();
-        parameterBuilder.addCorrelationId(correlationId);
-
-        if (!StringUtils.isEmpty(request.claims) || this.config.authOptions.clientCapabilities && this.config.authOptions.clientCapabilities.length > 0) {
-            parameterBuilder.addClaims(request.claims, this.config.authOptions.clientCapabilities);
-        }
-
-        return parameterBuilder.createQueryString();
+        return this.executePostToTokenEndpoint(authority.tokenEndpoint+"?dc=ESTS-PUB-WUS2-AZ1-TEST1", requestBody, headers, thumbprint);
```

Comment:
Was the dc information intentionally committed? Need to make sure to remember to remove it before merging to dev...

Reply:
I wonder if there's a better way we can provide test slices in the future that doesn't require changes to this code, as this isn't the first time we've had to do this and wont be the last

Reply:
We currently don't allow extraQueryParameters on the token endpoint, which is why this fix was added. I agree we should add a feature to hit test slices, I will look into making a PR for this.

Reply:
will remove before merging `broker` branch to dev.